src/mainboard/emulation/qemu-q35/src/main.rs

```diff
@@ -32,13 +25,10 @@ pub extern "C" fn _start(fdt_address: usize) -> ! {
 
     let w = &mut print::WriteTo::new(uart0);
 
-    let payload = &mut payload::StreamPayload { typ: payload::ftype::CBFS_TYPE_SELF, compression: payload::ctype::CBFS_COMPRESS_NONE, offset: 0, entry: 0x1000020 as usize, rom_len: 0 as usize, mem_len: 0 as usize, dtb: 0, rom: 0xff000000 };
+    let payload = &mut BzImage { low_mem_size: 0x80_000_000, high_mem_start: 0x1_000_000_000, high_mem_size: 0, rom_base: 0xff_000_000, rom_size: 0x100_000, load: 0x1_000_000, entry: 0x1_000_200 };
```

Comment:
Add TODO saying these values shouldn't be hard-coded.

Reply:
to pick this up from qemu is a substantial additional bit of work, so leave comment there too.